packages/mui-base/src/Portal/Portal.test.tsx

```diff
@@ -79,25 +79,25 @@ describe('<Portal />', () => {
         </Portal>
       </div>,
     );
-    const rootElement = document.querySelector('#test1');
-    expect(rootElement.contains(document.querySelector('.woofPortal1'))).to.equal(true);
-    expect(rootElement.contains(document.querySelector('.woofPortal2'))).to.equal(false);
+    const rootElement = document.querySelector<HTMLDivElement>('#test1');
+    expect((rootElement as HTMLDivElement).contains(document.querySelector('.woofPortal1'))).to.equal(true);
+    expect((rootElement as HTMLDivElement).contains(document.querySelector('.woofPortal2'))).to.equal(false);
   });
 
   it('should unmount when parent unmounts', () => {
     function Child() {
-      const containerRef = React.useRef();
+      const containerRef = React.useRef<HTMLDivElement>();
```

Comment:
When the ref is initialized with `null`, you can remove the cast in line 92.

```suggestion
      const containerRef = React.useRef<HTMLDivElement>(null);
```